app/components/Map.js

```diff
@@ -1,24 +1,79 @@
-import React, { Component, PropTypes, StyleSheet, Text, View  } from 'react-native';
+
+import React, { Component, PropTypes, StyleSheet, Text, View, Dimensions, AlertIOS  } from 'react-native';
+
+import MapView from 'react-native-maps';
+
 
 export default class Map extends Component {
   constructor(props) {
     super(props);
     this.state = {
+      position: {
+        latitude: 37.78825,
+        longitude: -122.4324,
+        latitudeDelta: 0.0922,
+        longitudeDelta: 0.0421,
+      },
+      lastPosition: undefined,
+      pins: [],
+      walker: undefined
     };
   }
+
+
+  componentDidMount() {
+    navigator.geolocation.getCurrentPosition(
+      (position) => {
+        var initialPosition = JSON.stringify(position);
+        this.setState({walker: initialPosition});
+      },
+      (error) => {
+        alert(error.message)
+      },
+      {enableHighAccuracy: true, timeout: 20000, maximumAge: 1000}
+    );
+
+    this.watchID = navigator.geolocation.watchPosition(
+      (position) => {
+        var lastPosition = JSON.stringify(position);
+        this.setState({lastPosition: position});
+      }
+    );
+
+  }
+
+  onRegionChange(region) {
+    this.setState({ position: region });
+  }
+
   render() {
     return (
-     <View style={[styles.container, { backgroundColor: 'transparent'}]}>
-        <Text style={styles.welcome}>
-          Welcome to React Native!
-        </Text>
-        <Text style={styles.instructions}>
-          To get started, edit index.ios.js
-        </Text>
-        <Text style={styles.instructions}>
-          Press Cmd+R to reload,{'\n'}
-          Cmd+D or shake for dev menu
-        </Text>
+      <View style={styles.container}>
+        <MapView
```

Comment:
You forgot to close the element like ---->  <MapView></MapView> 
Instead you just put a self closing one and that doesn't work since we will want to render Marker inside MapView later, and I'm not sure if that's the correct way to do it anyways. --->  <MapView />